crates/autopilot/src/run.rs

```diff
@@ -516,7 +516,8 @@ pub async fn run(args: Arguments) {
     );
 
     let liveness = Arc::new(Liveness::new(args.max_auction_age));
-    observe::metrics::serve_metrics(liveness.clone(), args.metrics_address);
+    let readiness = Arc::new(Some(AtomicBool::new(false)));
```

Comment:
Why does readiness contain an `Option`?

Reply:
So other services can pass `None` to serve_metrics() and opt-out from using the /ready endpoint that way.